verification/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -163,6 +163,13 @@ def run_build(litani, jobs):
     cmd = [str(litani), "run-build"]
     if jobs:
         cmd.extend(["-j", str(jobs)])
+    # Set a limit on the number of concurrent jobs based on CPUs
+    # https://github.com/aws/aws-encryption-sdk-c/pull/673
```

Comment:
I don't think there is a lot of value in pointing to the troublesome pull request.  I think a more helpful comment would be something like "Restrict concurrency during CI to a level that allows proofs requiring a lot of memory from consuming memory and causing other proofs to fail with 'out of memory'."